cluster/charts/crossplane/templates/deployment.yaml

```
@@ -239,3 +239,4 @@ spec:
       {{- if .Values.affinity }}
       affinity: {{ toYaml .Values.affinity | nindent 8 }}
       {{- end }}
+      dnsPolicy: {{ .Values.dnsPolicy }}
```

Maybe I'd prefer leaving the default "" 🤔
```suggestion
      {{- with .Values.affinity }}
      dnsPolicy: {{ . }}
      {{- end }}
```

---

Thank you for the feedback. I changed it assuming you meant `.Values.dnsPolicy` and not `.Values.affinity`.

---

Aha, yeah 😅 you got it right